src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java

```diff
@@ -27,6 +27,7 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
 import java.util.Map;
```

Comment:
I think we are save moving to Java 7. Even for JDK7 the public EOL is reached after April 2015:
http://www.oracle.com/technetwork/java/javase/downloads/eol-135779.html

Only moving to JDK8 would be a giant leap....
However we should include this info in the readme and I would suggest to include this in a major release (2.2.0) instead of a minor version (2.1.14).